index.html

```diff
@@ -14,6 +14,17 @@
   <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/main.css">
 
+  <link rel="stylesheet" type="text/css" href="./css/addtohomescreen.css">
+  <script src="./js/addtohomescreen.min.js"></script>
+  <script>
+      addToHomescreen();
+  </script>
+
+  <link rel="apple-touch-icon" sizes="57x57" href="./img/ios-icon-57x57.png" />
```

Comment:
And when previewing the SVG, it appears blank? Just a transparent image? Is that just me or did you upload the wrong thing?